tests/integration-tests/conftest.py

```diff
@@ -1381,18 +1381,23 @@ def _run_benchmarks(remote_command_executor, scheduler_commands, **kwargs):
 
 
 @pytest.fixture()
-def scheduler_plugin_configuration(request, region, upload_scheduler_plugin_definitions, scheduler=None):
-    scheduler_plugin = request.config.getoption("tests_config", default={}).get("scheduler-plugins", {}).get(scheduler)
-    scheduler_definition_url = upload_scheduler_plugin_definitions.get(scheduler, {}).get(region, {})
-    if scheduler_definition_url:
-        logging.info(
-            "Overriding scheduler plugin (%s) scheduler-definition to be (%s)",
-            scheduler,
-            scheduler_definition_url,
-        )
-        scheduler_plugin["scheduler-definition"] = scheduler_definition_url
+def scheduler_plugin_configuration(request, region, upload_scheduler_plugin_definitions):
+    try:
+        scheduler = request.getfixturevalue("scheduler")
+        if scheduler:
+            scheduler_plugin = request.config.getoption("tests_config", default={}).get("scheduler-plugins", {}).get(scheduler)
+            scheduler_definition_url = upload_scheduler_plugin_definitions.get(scheduler, {}).get(region, {})
+            if scheduler_definition_url:
+                logging.info(
+                    "Overriding scheduler plugin (%s) scheduler-definition to be (%s)",
+                    scheduler,
+                    scheduler_definition_url,
+                )
+                scheduler_plugin["scheduler-definition"] = scheduler_definition_url
 
-    return scheduler_plugin
+            return scheduler_plugin
+    except pytest.FixtureLookupError:
+        pass
```

Comment:
return None so that all functions paths have a return statement. optionally you could also only wrap `getfixturevalue` with try except

Reply:
thanks, updated